packages/manager/modules/adp/src/deploy/storage.html

```diff
@@ -0,0 +1,112 @@
+                <dt class="storage-list__term"><span data-translate="adp_deploy_storage_storage_per_node"></span> : </dt>
+                <dd class="storage-list__definition">
+                    <span>
+                        {{ ($ctrl.storage.hdfs_effective_storage * $ctrl.selectedCapability.hdfs_replication_factor) / $ctrl.nodesConfig.worker.count }}
+                    </span>
+                    <span data-translate="adp_common_unit_gb"></span>
+                </dd>
+                <hr class="storage-list__seperator"/>
+                <dt class="storage-list__term"><strong><span data-translate="adp_deploy_storage_total_storage"></span> :</strong></dt>
+                <dd class="storage-list__definition">
+                    <strong>
+                        <span>
+                            {{ $ctrl.storage.hdfs_effective_storage * $ctrl.selectedCapability.hdfs_replication_factor }}
+                        </span>
+                        <span data-translate="adp_common_unit_gb"></span>
+                    </strong>
+                    <button type="button"
+                            class="oui-popover-button"
+                            data-oui-popover="{{ ::'adp_deploy_storage_replication_factor_popover' | translate: {hdfc_replication_factor: $ctrl.selectedCapability.hdfs_replication_factor} }}">
+                    </button>
+                </dd>
+            </dl>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-6 col-xs-12">
+            <oui-field data-label="{{ ::'adp_deploy_storage_edge_node_label' | translate }}"
+                data-help-text="{{ ::'adp_deploy_storage_cluster_storage_helptext' | translate: {
+                    adp_version: $ctrl.adp.cluster_type,
+                    min_storage: $ctrl.storage.min_edge_node_storage,
+                    max_storage: $ctrl.storage.max_edge_node_storage
+                } }}"
+                data-size="m"
+                data-error-messages="{ pattern: ('adp_deploy_storage_invalid_entry' | translate) }">
+                <div class="oui-input-group">
+                    <input class="oui-input oui-input_number"
+                        type="number"
+                        id="hdfs_replication_factor"
+                        name="hdfs_replication_factor"
+                        required
+                        placeholder="{{ ::'adp_deploy_storage_set_quantity' | translate }}"
+                        data-ng-model="$ctrl.adp.edge_node_storage"
+                        data-ng-pattern="/^\d+$/"
+                        data-ng-min="{{ ::$ctrl.storage.min_edge_node_storage }}"
+                        data-ng-max="{{ ::$ctrl.storage.max_edge_node_storage }}">
+                    <button class="oui-button oui-button_icon-only oui-button_small-width"
+                        type="button"
+                        disabled>
+                        <span data-translate="adp_common_unit_gb"></span>
```

Comment:
You can apply `translate` directive directly on the parent element and get rid off that `span` element.

Reply:
Done